src/math/Box3.js

```diff
@@ -384,7 +384,7 @@ Object.assign( Box3.prototype, {
 
 		}
 
-		return ( min <= plane.constant && max >= plane.constant );
+		return ( min <= -plane.constant && max >= -plane.constant );
```

Comment:
Please insert a space after the signs. Otherwise the linter won't be happy...

Reply:
Thank you for your remind. Whitespaces added.